src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx

```diff
@@ -11,10 +11,12 @@ import { BitgoEnv, safeEnv } from '~/helpers';
 
 export function EvmCrossChainRecoveryBaseForm({
   formik,
+  isCustodyWallet,
 }: {
   formik: {
     setFieldValue: (a: string, b: string) => void;
   };
+  isCustodyWallet: boolean;
```

Comment:
can't we get this using useParams<'env' | 'wallet'> ?

Reply:
Started using useParams <'wallet'>